spec.html

```diff
@@ -26475,6 +26475,30 @@ <h1>Abstract Module Records</h1>
             </tr>
           </table>
         </emu-table>
+
+        <emu-clause id="sec-EvaluateModuleSync" type="abstract operation">
+          <h1>
+            <ins>
+              EvaluateModuleSync (
+                _module_: a Module Record,
+              ): either a normal completion containing ~unused~ or a throw completion
+            </ins>
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It synchronously evaluates _module_, provided that the caller guarantees that _module_'s evaluation will return an already settled promise.</dd>
+          </dl>
+
+          <emu-alg>
+            1. Assert: _module_ is not a Cyclic Module Record.
```

Comment:
This assertion is because non-Cyclic Module Records are guaranteed to be evaluated synchronously. The import defer proposal will relax this assertion to the one in https://tc39.es/proposal-defer-import-eval/#sec-EvaluateSync.